test/styles.css

```diff
@@ -7,3 +7,12 @@ body {
 .visual-diff {
 	margin-bottom: 30px;
 }
+.visual-diff-dark {
+	background: repeating-linear-gradient(
+		45deg,
+		#606dbc,
+		#606dbc 10px,
+		#465298 10px,
+		#465298 20px
+	);
```

Comment:
I'd like to use this striped background for other dark mode visual diffs that have some opacity to them. It'll help us catch regressions that accidentally break the opacity.

Reply:
Cool.  This might be better for the background that's used for the translucent button visual-diff too.

Reply:
Good call. I'll have follow-up PRs for dark mode button-icons so will include it with that.